Resources/Prototypes/Entities/Objects/Specific/Salvage/ore_bag.yml

```
@@ -26,3 +26,27 @@
         - ArtifactFragment
         - Ore
   - type: Dumpable
+
+- type: entity
+  name: borg ore bag
+  id: BorgOreBag
+  parent: OreBag
+  description: A robust bag for salvage specialists and miners alike to carry large amounts of ore. Magnetises any nearby ores when attached to a belt.
+  components:
+  - type: MagnetPickup
+  - type: Sprite
+    sprite: Objects/Specific/Mining/ore_bag.rsi
+    state: icon
+  - type: Item
+    size: Ginormous
+  - type: Storage
+    maxItemSize: Normal
+    grid:
+    - 0,0,9,5
+    quickInsert: true
+    areaInsert: true
+    whitelist:
+      tags:
+        - ArtifactFragment
+        - Ore
+  - type: Dumpable
```

these are also inherited, but remember to keep the certain part you're changing from the base

```suggestion
  - type: Storage
    grid:
    - 0,0,9,5
```

---

I believe that -type: Storage needs to stay because its grid is larger?